features/step_definitions/restroom_steps.rb

```diff
@@ -1,5 +1,5 @@
 Given(/^a restroom exists in Winnipeg$/) do
-  FactoryGirl.create(:restroom, {name: 'Winnipeg restroom', street: '91 Albert St.', city: 'Winnipeg', state: 'MB', country: 'Canada'}.merge(locations[:Winnipeg]))
+  FactoryBot.create(:restroom, {name: 'Winnipeg restroom', street: '91 Albert St.', city: 'Winnipeg', state: 'MB', country: 'Canada'}.merge(locations[:Winnipeg]))
```

Comment:
Since we have `config.include FactoryBot::Syntax::Methods`, we could just do `create(:restroom, ...)`, like is done elsewhere throughout the test suite. I personally prefer not including that, and being explicit instead by including `FactoryBot.` (though in this case, renaming the library results in more diff noise... I don't think they'll rename it again). That's outside the scope of this PR, though.

Reply:
Also, [I hate Winnipeg](https://www.youtube.com/watch?v=xLlsjEP7L-k).